samples/proxy-provider-node/README.md

```diff
@@ -0,0 +1,164 @@
+
+### On-behalf-of
+
+In on-behalf-of mode, the service uses the Microsoft identity platform's [on-behalf-of flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) to exchange the token sent by the calling client for a Microsoft Graph token. The next section gives instructions on how to register the app in Azure Active Directory.
+
+#### App registration
+
+This sample requires two app registrations. This is needed to take advantage of [combined consent](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow#default-and-combined-consent) with the on-behalf-of flow in the Microsoft identity platform. Create an app registration for:
+
+- The Graph Proxy service
+- The React front-end app
+
+##### Graph Proxy service
+
+1. Open a browser and navigate to the [Azure Active Directory admin center](https://aad.portal.azure.com). Login using a **personal account** (aka: Microsoft Account) or **Work or School Account**.
+
+1. Select **Azure Active Directory** in the left-hand navigation, then select **App registrations** under **Manage**.
+
+1. Select **New registration**. On the **Register an application** page, set the values as follows.
+
+    - Set **Name** to `Node.js Graph Proxy`.
+    - Set **Supported account types** to **Accounts in any organizational directory and personal Microsoft accounts**.
+    - Under **Redirect URI**, leave the value empty.
+
+1. Select **Register**. On the **Node.js Graph Proxy** page, copy the value of the **Application (client) ID** and **Directory (tenant) ID**. Save them, you will need them in the next step.
+
+1. Select **Certificates & secrets** under **Manage**. Select the **New client secret** button. Enter a value in **Description** and select one of the options for **Expires** and select **Add**.
+
+1. Copy the client secret value before you leave this page. You will need it in the next step.
+
+    > **IMPORTANT**
+    > This client secret is never shown again, so make sure you copy it now.
+
+1. Select **API permissions** under **Manage**, then select **Add a permission**.
+
+1. Select **Microsoft Graph**, then **Delegated permissions**.
+
+1. Select the following permissions, then select **Add permissions**.
+
+    - **Presence.Read** - this will allow the app to read the authenticated user's presence.
+    - **Tasks.ReadWrite** - this allows the app to read and write the user's To-Do tasks.
+    - **User.Read** - this will allow the app to read the user's profile and photo.
+
+    > **NOTE**
+    > These are the permissions required for the Microsoft Graph Toolkit components used in the client application. If you use different components, you may require additional permissions. See the [documentation](https://docs.microsoft.com/graph/toolkit/overview) for each component for details on required permissions.
+
+1. Select **Expose an API**. Select the **Set** link next to **Application ID URI**. Accept the default and select **Save**.
```

Comment:
Here I didn't see a `Set` link. Instead, I had to click on `Add a scope` which opened up the panel to accept the default link. 

![image](https://user-images.githubusercontent.com/711864/98321247-721ea380-1f99-11eb-81af-c5eff8a64c09.png)

Reply:
It's easy to miss it, but in your screenshot, where the disabled text box next to **Application ID URI** is now, there is originally a "Set" hyperlink.